network-api/networkapi/buyersguide/factory.py

```diff
@@ -197,9 +206,120 @@ class Meta:
+    easy_to_learn_and_use = Faker('boolean')
+    easy_to_learn_and_use_helptext = Faker('sentence')
+    first_published_at = Faker('past_datetime', start_date='-2d', tzinfo=timezone.utc)
+    last_published_at = Faker('past_datetime', start_date='-1d', tzinfo=timezone.utc)
+    slug = None
+
+
+class GeneralProductPageFactory(PageFactory):
+
+    class Meta:
+        model = GeneralProductPage
+
+    title = Faker('sentence')
+    camera_app = LazyFunction(get_extended_yes_no_value)
+    camera_device = LazyFunction(get_extended_yes_no_value)
+    microphone_app = LazyFunction(get_extended_yes_no_value)
+    microphone_device = LazyFunction(get_extended_yes_no_value)
+    location_app = LazyFunction(get_extended_yes_no_value)
+    location_device = LazyFunction(get_extended_yes_no_value)
+    personal_data_collected = Faker('sentence')
+    biometric_data_collected = Faker('sentence')
+    social_data_collected = Faker('sentence')
+    how_can_you_control_your_data = Faker('sentence')
+    data_control_policy_is_bad = Faker('boolean')
+    company_track_record = get_random_option(['Great', 'Average', 'Needs Improvement', 'Bad'])
+    track_record_is_bad = Faker('boolean')
+    track_record_details = Faker('sentence')
+    offline_capable = LazyFunction(get_extended_yes_no_value)
+    offline_use_description = Faker('sentence')
+    uses_ai = LazyFunction(get_extended_yes_no_value)
+    ai_uses_personal_data = LazyFunction(get_extended_yes_no_value)
+    ai_is_transparent = LazyFunction(get_extended_yes_no_value)
+    ai_helptext = Faker('sentence')
+    blurb = Faker('sentence')
+    company = Faker('company')
+    email = Faker('email')
+    live_chat = Faker('url')
+    phone_number = Faker('phone_number')
+    price = Faker('random_number', digits=3)
+    product_url = Faker('url')
+    twitter = '@TwitterHandle'
+    worst_case = Faker('sentence')
+    first_published_at = Faker('past_datetime', start_date='-2d', tzinfo=timezone.utc)
+    last_published_at = Faker('past_datetime', start_date='-1d', tzinfo=timezone.utc)
+    slug = None
+
+
+class ProductPagePrivacyPolicyLinkFactory(DjangoModelFactory):
+
+    class Meta:
+        model = ProductPagePrivacyPolicyLink
+
+    label = Faker('sentence')
+    url = Faker('url')
+
+
 def generate(seed):
     reseed(seed)
 
+    print('Generating PNI Homepage')
+    pni_homepage = BuyersGuidePageFactory.create(
+        parent=Homepage.objects.first(),
+        title='* Privacy not included',
+        slug='privacynotincluded-new',
+    )
+
+    print('Generating 100 ProductPages')
+    for i in range(50):
+        # Create 50 GeneralProductPages with Privacy Link Orderables
+        general_page = GeneralProductPageFactory.create(
+            parent=pni_homepage,
+        )
+        fake_privacy_policy = ProductPagePrivacyPolicyLinkFactory(
+            page=general_page
+        )
+        general_page.privacy_policy_links.add(fake_privacy_policy)
+        general_page.save_revision().publish()
```

Comment:
This looks like it's missing setting zero-or-more `related_products`.

Reply:
Added 3 related products per page